client/src/components/App.js

```diff
@@ -31,11 +31,20 @@ const App = (props) => {
       <TopBar user={currentUser} />
       <Switch>
         <Route exact path="/">
-          <SocialMediaIndexPage />
+        <SocialMediaList className="grid-container" />
```

Comment:
hmmm better, but still weird

- this `className` is actually just a prop...and its not being used. If you want this className to be applied, just manually add it directly to an element inside of SocialMediaList
- also indent in this line